akka-http-tests/src/test/java/akka/http/javadsl/server/directives/ParameterDirectivesTest.java

```diff
@@ -31,6 +31,11 @@ public void testStringParameterExtraction() {
       .run(HttpRequest.create().withUri("/abc?stringParam=john"))
       .assertStatusCode(200)
       .assertEntity("john");
+
+    route
+      .run(HttpRequest.create().withUri("/abc?stringParam=a=b"))
+      .assertStatusCode(400)
+      .assertEntity("The request content was malformed:\nThe request's query string is invalid: Illegal query: Invalid input '=', expected '+', query-char, 'EOI', '&' or pct-encoded (line 1, column 14): stringParam=a=b\n             ^");
```

Comment:
would be nice if we can combine these three phrases "The request content was malformed:\nThe request's query string is invalid: Illegal query"

also there should be a newline before `stringParam=a=b\n` so that the error marker makes sense (or just remove it?)